db/schema.rb

```diff
@@ -48,7 +48,6 @@
     t.string   "listing_id"
     t.string   "document_type"
     t.integer  "user_id"
-    t.integer  "preference"
```

Comment:
@kaptron Is this schema change right?

Reply:
@creativecoder ah yeah didn't leave an explanation on this, but yeah this was a correction for a field that's already been removed in MF (i.e. if you run all the migrations in order it is removed). I had fudged it to leave the field in my local db for easier switching between MF and pre-MF, so this is just setting it back to the way it should be.